front/src/views/CheckoutView/index.js

```diff
@@ -0,0 +1,157 @@
+import CheckoutCard from "components/CheckoutComponents/CheckoutCard";
+import CheckoutRow from "components/CheckoutComponents/CheckoutRow";
+
+import {
+  Grid,
+  makeStyles,
+  createStyles,
+  Divider,
+  Typography,
+  Box,
+} from "@material-ui/core";
+import { Title } from "features/carts/CartStyle";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      padding: "130px 60px",
+      height: "100%",
+      flexDirection: "row",
+    },
+    cardsRoot: {
+      padding: "20px",
+      marginRight: "20px",
+    },
+    orderTitle: {
+      fontWeight: "bold",
+      textAlign: "center",
+    },
+    orderRoot: {
+      paddingTop: "20px",
+      maxWidth: "270px",
+      [theme.breakpoints.down("lg")]: {
+        width: "260px",
+        justifyContent: "center",
+        maxWidth: "unset",
+      },
+      [theme.breakpoints.down("md")]: {
+        width: "100%",
+        justifyContent: "center",
+      },
+    },
+    orderDivider: {
+      backgroundColor: "Grey",
+      marginTop: "10px",
+      marginBottom: "10px",
+      height: "1px",
+    },
+    checkoutWrapper: {
+      flexDirection: "column",
+    },
+  })
+);
+
+function CheckoutView() {
+  const classes = useStyles();
+  var itemCount = 2;
+
+  return (
+    <Grid container className={classes.root}>
```

Comment:
`<Grid container className={classes.root}>`
     ` <Grid item xs={6} className={classes.cardsRoot}>`
      `  <Grid container spacing={6} direction="column">`
       `   <Grid item>`
mmm cia apsikrauni, labai pamilai grid'a :D 
gali prasitrint antra grid container ir grid item ir pasikeist i boxus, bus paprasciau (boxuose display flex naudok ir isspres tavo problema ir irgi gali flexDirection: column uzsimest, tas ant to bus.